src/api.js

```diff
@@ -0,0 +1,14 @@
+
+// eslint-disable-next-line max-len
+const BASE_URL = 'https://mate-academy.github.io/react_people-table/api/people.json';
+
+export function getPeople() {
+  return fetch(BASE_URL)
+    .then((response) => {
+      if (!response.ok) {
+        throw new Error(`${response.status} - ${response.statusText}`);
+      }
+
+      return response.json();
+    });
+}
```

Comment:
And what about the second part of the task?
`  - Find a `mother` and a `father` by `motherName` and `fatherName` and add them to the person for future use`